bin/webpack.js

```diff
@@ -29,35 +29,59 @@ try {
 	webpackCliInstalled = false;
 }
 
+let webpackCommandInstalled = false;
```

Comment:
Instead of duplicating this, create an array of all `CLI`s and change the existing code to iterate.